### PR TITLE
fix(ssh): still display sshkeys if one is in error

### PR DESCRIPTION
**Pull request by marie-j**
Related DTRUN-3865
MANAGER-3696

### CODE REVIEW

File: client/app/billing/autoRenew/ssh/user-ssh.controller.js [hunk 1]
```diff
@@ -63,7 +63,8 @@ export default class UserAccountSshCtrl {
       .then((sshKeys) => {
         this.sshKeyList = sshKeys;
       })
-      .catch((err) => {
+      .catch(([sshKeys, err]) => {
```
Comment (FredericEspiau) [inline client/app/billing/autoRenew/ssh/user-ssh.controller.js]: 🤔

File: client/app/billing/autoRenew/ssh/user-ssh.service.js [hunk 2]
```diff
@@ -22,10 +22,18 @@ angular.module('UserAccount').service('UseraccountSshService', [
           promises.push(this.getCloudSshList());
       }
 
-      return $q.all(promises).then((data) => {
-        const keys = _.flatten(_.values(data));
-        return _.sortBy(keys, key => key.keyName.toLowerCase());
-      });
+      return $q.allSettled(promises)
+        .then(sshKeys => [sshKeys])
```
Comment (FredericEspiau) [inline client/app/billing/autoRenew/ssh/user-ssh.service.js]: It's not already an array ?

File: client/app/billing/autoRenew/ssh/user-ssh.service.js [hunk 3]
```diff
@@ -22,10 +22,18 @@ angular.module('UserAccount').service('UseraccountSshService', [
...
+        .catch(([sshKeys, error]) => (_.isArray(sshKeys) ? [sshKeys, error] : [[], error]))
+        .then(([sshKeys, error]) => {
```
Comment (FredericEspiau) [inline client/app/billing/autoRenew/ssh/user-ssh.service.js]: Won't it set the second key in `error` ?

Comment (marie-j) [reply client/app/billing/autoRenew/ssh/user-ssh.service.js]: You might be right, I think I misunderstood the way `allSettled` works. checking right away

Comment (marie-j) [reply client/app/billing/autoRenew/ssh/user-ssh.service.js]: I misunderstood `allSettled` in that context. Now fixed

**Pull request merged**